extensions/wikia/RTE/js/plugins/placeholder/plugin.js

```diff
@@ -22,6 +22,16 @@ CKEDITOR.plugins.add('rte-placeholder',
 			// get all placeholders (template / magic words / double underscores / broken image links)
 			var placeholders = RTE.tools.getPlaceholders();
 
+			// if placeholder does not have content, render green puzzle
+			placeholders.each(function (p) {
+				var $placeholder = $(placeholders.get(p));
+				console.log($placeholder);
+				if ( $.trim($placeholder.html().replace(/[\u200B]/, '')) === '') {
```

Comment:
Let's add a comment that we need to get rid of the empty spaces before comparison